meetings/cpansec-minutes-2025-03-26.md

```diff
@@ -0,0 +1,216 @@
+    - Also, something we can raise funding for
+
+
+### [Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1/views/2)
+
+#### CPAN::Meta::Spec, Requirements and PURLs
+- [ ] @sjn - picking up CPAN::Meta::Spec after FOSDEM
+- [ ] @sjn - need to push GDT's PURL update to it's spec
+   - @stigtsp - Some delays in adding it to CVEs
+
+#### SBOM/Metadata on CPAN
+- [ ] @tux - we need tools for working with SBOMs. *correct* SBOMs.
+   - @sjn - @gdt has written some modules for generating CycloneDX 1.6 documents from CPAN Metadata: [SBOM::CycloneDX](https://github.com/giterlizzi/perl-SBOM-CycloneDX) and [App::CPAN::SBOM](https://github.com/giterlizzi/perl-App-CPAN-SBOM)
+   - [ ] @sjn and @tux will check them out and comment on usability
+
+#### CycloneDX 1.7 Sustainability fields
+- @sjn - Ongoing: project- & ecosystem-supplied status fields.
+   - @sjn - Certain ecosystem-sourced project statuses are taken out of scope for now; They may be added at a later version of the spec.
+
+
+### [Compliance, Guidance & Privacy](https://github.com/orgs/CPAN-Security/projects/9/views/2)
+
+#### CPAN Author's Security Policy Guidelines
+- [ ] @stigtsp - Minimal (TLDR) version of policy wanted
+- [ ] @tux - Team document template wanted; Help needed
+   - [ ] @timlegge has started on something; happy to discuss with @tux; PR in the works
+- [ ] @sjn - Alternative documents needed for non-Europeans (requested by jnap)
+   - @sjn - can help with a CRA perspective
+   - @sjn - ORC WG FAQ may help
+
+### [Security Information & Outreach](https://github.com/orgs/CPAN-Security/projects/12/views/3)
+
+#### Recruitment
+- [x] @sjn - look at the next events for opportunities
+   - @sjn will attend GPW and Foss North partly for this reason.
+
+#### Perl Toolchain Summit 2025
+- [ ] @sjn - Create document on what CPANSec-relevant issues need to be decided on/discussed at PTS
+    - Everyone, please add tasks!
+    - [ ] @stigtsp - Better tooling for CNA CVE
+    - [ ] @stigtsp - Show CVEs on MetaCPAN
+    - [ ] @stigtsp - Better feed for Test::CVE and friends
+
+#### New Guide needed: PackageURL
+- [ ] @sjn - We need a comprehensive PackageURL guide that answers/covers all relevant use cases
+    - Non-perl project uses perl executable provided by packaging system // plenv // build-perl // ActiveState // etc.
+    - Non-perl project uses a tool provided on CPAN
+    - Non-perl project uses a vendored/included tool written in Perl
+    - Perl project not on CPAN uses a CPAN module found on CPAN
+    - Perl project found on CPAN uses a CPAN module found on CPAN
+    - Perl project not found on CPAN uses a CPAN module not found on CPAN, but a DarkPAN
+    - etc.
+
+#### StarJacking Study
+- [ ] @thibaultduponchelle - CPAN StarJacking study
+   - [x] @garu @thibaultduponchelle - Checked riverjacking algo with Neil for faking high river score with one single author
+   - [ ] Ready. Merge?
```

Comment:
I was under the impression that this was not ready for disclosure yet

Reply:
Some parts were removed, iirc. @thibaultduponchelle?

Reply:
Yes, better remove this line

Reply:
I updated